lib/src/models/severity.dart

```diff
@@ -0,0 +1,51 @@
+/// Enum class for severity
+/// 
+/// Used when reporting issues
+class Severity {
+  /// Programming error
+  ///
+  /// This indicates severe issue like memory leak etc
+  /// The issue is certain
```

Comment:
Correction: `Indicates a severe issue like a memory leak, etc`

And I'd suggest to drop `The issue is certain`